Source/AtomicEditor/Application/AEEditorCommon.cpp

```diff
@@ -150,4 +155,38 @@ void AEEditorCommon::Stop()
 #endif
 }
 
+
+bool AEEditorCommon::ReadPreferences()
+{
+
+    FileSystem* fileSystem = GetSubsystem<FileSystem>();
```

Comment:
Does the ResourceCache subsystem exist here?  If so, can we use the GetTempResource method?

https://github.com/AtomicGameEngine/AtomicGameEngine/blob/master/Source/Atomic/Resource/ResourceCache.h#L142

I am not sure if this is wanting a file in the resource dirs, or if you give it an absolute path, it will load it